R/dynamic.R

```diff
@@ -56,27 +58,34 @@
 #' @export
 jekyll = function(
   dir = '.', input = c('.', '_source', '_posts'), output = c('.', '_posts', '_posts'),
-  script = c('Makefile', 'build.R'), serve = TRUE, ...
+  script = c('Makefile', 'build.R'), serve = TRUE, bundle = FALSE, ...
 ) {
   baseurl = jekyll_config(dir, 'baseurl', '')
   destination = jekyll_config(dir, 'destination', '_site')
-  jekyll_build = function() {
-    if (system2('jekyll', 'build') != 0) stop('Failed to run Jekyll')
+  jekyll_build = function(bundle) {
```

Comment:
You can remove the `bundle` argument. R knows where this object comes from. Same thing for the `jekyll_build()` calls below.